### DISCUSSION TITLE
Only apply query strings when specified

### DISCUSSION THREAD

**Discussion by lairg99** [livewire/livewire#4920] [1. Help] [answered]
Is it a desired behavior that query strings are applied even though they have not been declared in the $queryString property?

In my case I have two components, each with a search property. But only one has the search property defined as a query string. After a refresh, Livewire applies the query string to both components.

Currently I use aliases as work around.

**Comment by joshhanley** [comment, marked as answer]
@lairg99 hmm yeah that doesn't sound right. I wonder why it would be applying the search value to both components?

**Reply by lairg99** [reply]
Hey, sorry! There was unfortunate side effect in my application. Everything works fine!

**Reply by joshhanley** [reply]
No worries! Glad you got it sorted!
